internal/databases/mongo/binary/mongod.go

```diff
@@ -0,0 +1,128 @@
+package binary
+
+import (
+	"context"
+	"github.com/wal-g/wal-g/internal"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+const adminDB = "admin"
+
+type MongodService struct {
```

Comment:
What is the difference between `MongodService` and `MongoClient`? Maybe state explicitly that it is a wrapper and embed the mongo.Client into it? For example:


```
type MongoClientWrapper struct {
ctx context.Context
mongo.Client
}
```

Reply:
I want to incapsulate work with mongo server in this Service "class"